ietf/group/views.py

```diff
@@ -1354,6 +1354,85 @@ def group_menu_data(request):
     return JsonResponse(groups_by_parent)
 
 
+@cache_control(public=True, max_age=30 * 60)
+@cache_page(30 * 60)
+def group_stats_data(request, years="3", only_active="1"):
```

Comment:
Can we pull these out of the signature for now? If not, could we at least make only_active a Boolean?